application/controllers/Station.php

```diff
@@ -119,12 +119,23 @@ public function copy($id)
 			$data['my_station_profile']->station_profile_name = '';
 
 			if ($this->form_validation->run() == FALSE) {
+				// [eQSL default msg] GET from user options (same as edit()) //
+				$this->load->model('user_options_model');
```

Comment:
you don't need to load the model, as it is in the autoloader
```suggestion
```

Reply:
i delete this load model on create, update and copy